lib/capybara/spec/session/visit_spec.rb

```diff
@@ -17,6 +17,17 @@
     expect(@session).to have_content('Another World')
   end
 
+  # TODO: This test passes on racktest and selenium - although I think all it's really testing
+  # is that scheme defaults to http if not specified as anything else
+  # it "should fetch a response when absolute URI doesn't have a scheme", focus: true do
+  #   # Preparation
+  #   @session.visit('/')
+  #   root_uri = URI.parse(@session.current_url)
+  #
+  #   @session.visit("//#{root_uri.host}:#{root_uri.port}/foo")
```

Comment:
Do you know a use case for this feature?
@dstosik wanted to use it to load relative page links but that's not what `visit` is intended to do.